### PR TITLE
[UIPO-1] Project skeleton

### PR DESCRIPTION
**Pull request by TorskyM**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,26 @@
+### PROJECT TheSHOP
 
+For development, you will need Node.js installed on your environement. Please use the appropriate Editorconfig plugin.
+
+### Install Node
+
+You will need to use a Terminal.
+
+Type next command in Terminal: ```install node```
```
Comment (Nazar-bnm) [inline README.md]: Is it works ?

File: README.md [hunk 2]
```diff
@@ -1 +1,26 @@
+### PROJECT TheSHOP
 
+For development, you will need Node.js installed on your environement. Please use the appropriate Editorconfig plugin.
+
+### Install Node
+
+You will need to use a Terminal.
```
Comment (Nazar-bnm) [inline README.md]: Please add which version of the node we should use and how we can check it

File: README.md [hunk 3]
```diff
@@ -1 +1,26 @@
...
+
+Type next command in Terminal: ```install node```
+Then you need to install npm dependencies by typing: ```npm install``` or ```npm i```
+
+
+### Start & watch
+```npm run scss``` - start node-sass compiler
```
Comment (Nazar-bnm) [inline README.md]: why should we run the `scss` compiler? It should works automatically.

File: dist/index.html [hunk 4]
```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
```
Comment (Nazar-bnm) [inline dist/index.html]: dist should be ignored by git

File: package.json [hunk 5]
```diff
@@ -1,28 +1,61 @@
 {
   "name": "redux-simple-starter",
   "version": "1.0.0",
-  "description": "Simple starter package for Redux with React and Babel support",
+  "eslintConfig": {
+    "extends": "react-app"
+  },
+  "description": "My webpack project",
```
Comment (Nazar-bnm) [inline package.json]: Please change name and description to  more suitable

File: src/app/index.js [hunk 6]
```diff
@@ -1,7 +1 @@
-import React from 'react';
-import ReactDOM from 'react-dom';
-import App from './components/app';
-
-ReactDOM.render(
-    <App />
-  , document.querySelector('#root'));
+console.log("Hello World");
```
Comment (Nazar-bnm) [inline src/app/index.js]: Where do you connect React to the project?

**Review by Nazar-bnm — CHANGES_REQUESTED** [review]

File: package.json [hunk 7]
```diff
@@ -1,28 +1,62 @@
...
+    "style-loader": "^1.0.0",
+    "webpack": "^4.40.2",
+    "webpack-cli": "^3.3.9",
+    "webpack-dev-server": "^3.8.1"
   },
   "dependencies": {
+    "@webpack-cli/init": "^0.2.2",
+    "axios": "^0.19.0",
     "babel-preset-stage-1": "^6.1.18",
+    "eslint-config-standard-jsx": "^8.1.0",
```
Comment (Hollister009) [inline package.json]: Shouldn't it be in `devDependencies`?

File: package.json [hunk 8]
```diff
@@ -1,32 +1,71 @@
 {
   "name": "redux-simple-starter",
   "version": "1.0.0",
-  "description": "Simple starter package for Redux with React and Babel support",
+  "eslintConfig": {
+    "extends": "react-app"
+  },
+  "engines": {
+    "node": ">=12.0.0",
```
Comment (Hollister009) [inline package.json]: I'm not sure about this versioning 🤔maybe `"node": ">=10.x"`

File: src/app/components/app.js [hunk 9]
```diff
@@ -1,8 +1,13 @@
+/* eslint-disable require-jsdoc */
```
Comment (Hollister009) [inline src/app/components/app.js]: remove this comments

File: src/app/components/app.js [hunk 10]
```diff
@@ -1,8 +1,13 @@
...
+/* eslint-disable no-unused-vars */
+
```
Comment (Hollister009) [inline src/app/components/app.js]: Please remove unnecessary comments

File: src/app/scss/main.scss [hunk 11]
```diff
@@ -0,0 +1,12 @@
...
+
+    &-route {
+        scroll-behavior: auto;
+    }
+}
+
+body {
+    background-color: red;
+    font-size: 9000px;
+}
```
Comment (Hollister009) [inline src/app/scss/main.scss]: Delete test styles

File: src/app/index.js [hunk 12]
```diff
@@ -1,7 +1,8 @@
+/* eslint-disable no-unused-vars */
 import React from 'react';
 import ReactDOM from 'react-dom';
 import App from './components/app';
-import { Provider } from 'react-redux';
+import {Provider} from 'react-redux';
```
Comment (Hollister009) [inline src/app/index.js]: Why this change even here?

File: src/app/components/app.js [hunk 13]
```diff
@@ -1,8 +1,13 @@
...
 import CounterContainer from '../containers/CounterContainer';
 
 function App() {
-  return <CounterContainer/>;
+  return (
+    <CounterContainer/>
+  );
 }
 
 export default App;
```
Comment (Hollister009) [inline src/app/components/app.js]: I believe this file should be not included in changes

**Review by Hollister009 — CHANGES_REQUESTED** [review]
Needs a little cleanup. Remove `app.js` and `main.scss` from PR

**Review by Sviatoslav-Paliichuk — APPROVED** [review]

**Pull request merged**